ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -124,6 +130,9 @@ class PhaserUnit extends PhaserAnimatedEntity {
 		attrs: AttributeData[]
 	}): void {
 		console.log('PhaserUnit render-attributes', data); // TODO remove
+		const attributesContainer = this.attributesContainer = this.scene.add.container(0,
```

Comment:
`attributesContainer` should be created once in constructor, instead of creating new container every time attributes update. this code will create huge number of new containers and could potentially crash games due to memory leak